bot/exts/utils/ping.py

```diff
@@ -0,0 +1,57 @@
+import socket
+from datetime import datetime
+
+import aioping
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Emojis, URLs
+
+DESCRIPTIONS = (
+    "Command processing time",
+    "Python Discord website latency",
+    "Discord API latency"
+)
+ROUND_LATENCY = 3
+
+
+class Latency(commands.Cog):
+    """Getting the latency between the bot and websites."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
```

Comment:
A ping command isn't exactly something that is of any use in any other channel than bot commands, we probably should restrict it to only that channel

Reply:
Staff roles and the bot command channel have been whitelisted.